pkg/controlCenter/control_center.go

```diff
@@ -29,13 +29,22 @@ func CreateControlCenter(cityPointer *city.City) ControlCenter {
 				continue
 			}
 
-			c.routesBetweenNodes[tramStopPair] = c.GetShortestPath(firstStop.ID, secondStop.ID)
+			c.routesBetweenNodes[tramStopPair] = c.getShortestPath(firstStop.ID, secondStop.ID)
 		}
 	}
 	return c
 }
 
-func (c *ControlCenter) GetShortestPath(sourceID, destID uint64) []*city.GraphNode {
+func (c *ControlCenter) GetRouteBetweenNodes(sourceID, destID uint64) (path []*city.GraphNode) {
```

Comment:
As we're defining the path variable inside the code, I believe just a result type annotation will do.
```suggestion
func (c *ControlCenter) GetRouteBetweenNodes(sourceID, destID uint64) []*city.GraphNode {
```

Reply:
I couldn't add this to batch but changes it in code.